pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -228,13 +228,13 @@ func (m *manager) RemoveContainer(containerID string) error {
 	// if error appears it means container entry already does not exist under the container map
 	podUID, containerName, err := m.containerMap.GetContainerRef(containerID)
 	if err != nil {
-		klog.Warningf("[memorymanager] Failed to get container %s from container map error: %v", containerID, err)
+		klog.ErrorS(err, "Failed to get container from container map", "containerID", containerID)
```

Comment:
You have quite a few Warningf->ErrorS - shouldn't they be InfoS??

Can you please check throughout

Reply:
I think is correct to migrate Warningf to Info, but in this case should not be this an error log level?

Reply:
moving it to `klog.InfoS(...,  "err", err)`, just looks odd to me

Reply:
`InfoS` is the correct level for warnings. See the migration guide: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent

Reply:
@ehashman yes, the point is the Warning is already being wrong here.. but changed according to the migration guide